archive/cpio_old_le.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: cpio_old_le
+  title: cpio archive, old binary variant, little-endian
+  file-extension: cpio
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII

```

Comment:
We could take a look how cpio libraries work and find out which encodings are commonly used, but on the other hand that would be more effort we can spend on this format.

What we can do is support a WebIDE (/ksv) extension which gives a hint about that this field should be visualized as a string with and maybe give a best effort encoding (eg. UTF8) for visualizing (for example: `-webide-display-as: string`, `-webide-encoding: utf8`). What do you think, is it a good idea or not?